src/catalyst-toolbox/snapshot-lib/src/registration.rs

```diff
@@ -2,6 +2,9 @@ use jormungandr_lib::crypto::account::Identifier;
 use jormungandr_lib::interfaces::Value;
 use serde::{de::Error, Deserialize, Serialize};
 
+const MAINNET_PREFIX: &'static str = "addr";
+const STAKE_PREFIX: &'static str = "stake";
```

Comment:
We should consider the case of a testnet address as well where the prefix is `addr_test`